docs/src/index.md

```diff
@@ -181,8 +185,20 @@ linear (generic function with 2 methods)
 julia> grads = gradient(() -> sum(linear(x)), Params([W, b]))
 Grads(...)
 
+# Apply gradients to model parameters
 julia> grads[W], grads[b]
 ([0.652543 … 0.683588], [1.0, 1.0])
 ```
+Unlike with explicit gradients, in order to see implicit gradients one needs to do:
+
+```julia
+julia> grads.grads
+IdDict{Any, Any} with 5 entries:
+  [0.467471 0.597815 … 0.678126 … => [0.579671 0.215381 … 0.635058 0.623832; 0.579671 0.215381 … …
+  :(Main.x)                       => [1.3377, 0.930234, 0.499161, 1.33827, 1.37791]
+  :(Main.W)                       => [0.579671 0.215381 … 0.635058 0.623832; 0.579671 0.215381 … …
+  [0.106308, 0.705531]            => 2-element Fill{Float64}: entries equal to 1.0
+  :(Main.b)                       => 2-element Fill{Float64}: entries equal to 1.0
+```
```

Comment:
We don't want to point people to `grads.grads` cause it contains extraneous info (it's quick for debugging, but not recommended). The suggestion below is the recommended way.

Can you add the printed out text at the prompt?

```suggestion
To inspect the `Grads(...)` object returned for implicit parameters, you can index it using the parameters passed to `Params`:

```julia
julia> [grads[p] for p in [W, b]]
```
```